docfx.json

```diff
@@ -155,6 +169,7 @@
       },
       "ms.author": {
         "_csharplang/spec/*.md": "wiwagn",
+        "_csharplang/proposals/csharp-7.0/*.md": "wiwagn",
```

Comment:
you could perhaps just change the previous line to 
"_csharplang/**/**.md": "wiwagn",

Reply:
@mairaw I want to wait on that until the next updates (7.1 or 7.2). I think that's where it's going, but I need to understand what other content from the dependent repo could be accessed through docs. (It's all public, so it's not a danger, but it could mislead customers to include "rejected" or other proposals that aren't delivered yet.)